net.sourceforge.SDL-Ball.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: net.sourceforge.SDL-Ball
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: sdl-ball
+rename-icon: sdl-ball
+rename-desktop-file: sdl-ball.desktop
+rename-appdata-file: sdl-ball.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/SDL/SDL_image-1.2.12.json
+  - shared-modules/SDL/SDL_mixer-1.2.12.json
+  - shared-modules/SDL/SDL_ttf-2.0.11.json
```

Comment:
Have you considered using https://github.com/libsdl-org/sdl12-compat ?

Reply:
No, that won't be needed when https://github.com/DusteDdk/SDL-Ball/pull/8 lands.

Reply:
It hasn't landed for 2 years. I'd rather see if sdl12-compat works than hope for that to be merged.

Reply:
It works and it is so much better with a multi monitor setup. Great idea!